server/build_event_protocol/event_parser/event_parser.go

```diff
@@ -248,7 +249,10 @@ func FillInvocationFromEvents(buildEvents []*inpb.InvocationEvent, invocation *i
 		fillInvocationFromBuildMetadata(buildMetadatum, invocation)
 	}
 
-	buildDuration := time.Duration((endTimeMillis - startTimeMillis) * int64(time.Millisecond))
+	buildDuration := time.Duration(int64(0))
```

Comment:
We could use the current timestamp if `endTimeMillis` is not yet available, but I wasn't sure if we'd run into timezone issues when getting the current time.

Reply:
unix timestamps are timezone-independent, so it should be OK to use Now() to fill in a missing end time.

Reply:
Done